services/ui-src/src/components/fields/TextField.tsx

```diff
@@ -88,13 +92,11 @@ export const TextField = ({
   const parsedHint = hint && parseCustomHtml(hint);
   const nestedChildClasses = nested ? "nested ds-c-choice__checkedChild" : "";
   const labelClass = !label ? "no-label" : "";
-  const labelText = styleAsOptional ? labelTextWithOptional(label) : label;
+  const labelText =
+    label && styleAsOptional ? labelTextWithOptional(label) : label;
```

Comment:
Question: As it currently stands, this code will make it so if there's no label, there will also be no "(optional)" on a field. That's probably what we want, but we should double-check with design since that's a bit of an edge case. Will you please reach out to design and see what they want to do if there's no label text? This happens a fair amount with things like nested text boxes under "Other, specify" and similar.